.gitignore

```diff
@@ -13,7 +13,7 @@ web/css/
 web/media
 web/assetic
 build/
-vendor/
+/vendor/
```

Comment:
if we use bower to install jquery and bootstrap, i think we should not commit those files to git. if that does not work, i would prefer loading the files through a cdn url rather than managing them locally.